llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.cpp

```diff
@@ -85,6 +85,37 @@ void RISCVTargetELFStreamer::finishAttributeSection() {
                           ELF::SHT_RISCV_ATTRIBUTES, AttributeSection);
 }
 
+void RISCVTargetELFStreamer::emitGNUProgramProperties(unsigned Flags) {
```

Comment:
Change the name of `Flags` to `AndFlags`: Although currently only the `AND` flag is used, the `.note.gnu.property` section actually defines more flags besides the `AND` one, so if you're emitting the whole section with this function (as indicated by the function name), you should be clear about what flag of the section you're emitting.

Reply:
I should move emit program properties to TargetStreamer.
Thanks,